### ISSUE TITLE
Messages prop is not working

### ISSUE THREAD

**Issue by luisccf** [kenshoo/react-multi-select#150]
## Expected Behavior
Messages prop should be used to render the right messages

## Actual Behavior
Only `searchPlaceholder` message works

## Steps to Reproduce the Problem

  1. Pass messages prop to MultiSelect component

## Specifications

  - Version: 1.1.1
  - Platform: macOS

Here is my (simplified) code:
```javascript
import React from 'react';
import MultiSelect from '@kenshooui/react-multi-select';

const messages = {
  searchPlaceholder: 'Pesquisar...',
  noItemsMessage: 'Selecione um ou mais itens na coluna da esquerda',
  noneSelectedMessage: 'Nenhum selecionado',
  selectedMessage: 'Selecionado',
  selectAllMessage: 'Selecionar todos',
  clearAllMessage: 'Remover todos',
};

const InputMultiSelect = () => <MultiSelect messages={messages} />;

export default InputMultiSelect;
```
<img width="1082" alt="Captura de Tela 2019-05-16 às 10 55 57" src="https://user-images.githubusercontent.com/14101365/57859393-369e8100-77c9-11e9-8d25-7cdc366a8a34.png">

**Comment by stepovat** [comment]
hi @luisccf , thanks for reaching this out to us!
@MaksimDrobchak is on it, we will provide fix today

**Comment by stepovat** [comment]
@luisccf the issue is now corrected in v1.1.2
thanks for participation
